packer/opensuse/variables.pkr.hcl

```diff
@@ -31,3 +31,20 @@ variable "opensuse" {
     }
   }
 }
+
```

Comment:
IDK, but can we just hardcode the iso_checksum manually (without the `file:`) and wait until they fix?

Reply:
yes, we could also use openSUSE-Leap-15.6-Minimal-VM.x86_64-15.6.0-Cloud-Build13.356.qcow2 as the iso, which is the file the checksum points to. However the issue has been reported 3 weeks ago, and we could end up in a situation where a new build is pushed and breaks the build process again because the checksum is outdated.